cron_scripts/inactive_idle_accounts.pl

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Getopt::Long;
+use Data::Dumper;
+use JSON;
+use File::Slurp;
```

Comment:
nit: import only read_file by doing this (FWIW, neither JSON nor File::Slurp are core Perl modules):
```
use File::Slurp qw(read_file);
```

Reply:
Done